packages/react-native/package.json

```diff
@@ -35,7 +35,9 @@
     "style-dictionary": "3.9.1"
   },
   "peerDependencies": {
-    "aws-amplify": "^6.3.2",
+    "@react-native-async-storage/async-storage": "*",
```

Comment:
This only used for a mock that is not bundled in _dist_, can it be listed as a dev dependency?

Reply:
It's listed as a devDependency for `@aws-amplify/react-native`, in that case is our documentation incorrect for telling people to install it separately?